internal/pkg/manifest/backend_svc.go

```diff
@@ -128,8 +130,9 @@ func newDefaultBackendService() *BackendService {
 		BackendServiceConfig: BackendServiceConfig{
 			ImageConfig: ImageWithPortAndHealthcheck{},
 			TaskConfig: TaskConfig{
-				CPU:    aws.Int(256),
-				Memory: aws.Int(512),
+				CPU:      aws.Int(256),
+				Memory:   aws.Int(512),
+				Platform: PlatformArgsOrString{},
```

Comment:
no need to define it here now since it's no longer a pointer, the zero value `PlatformArgsOrString{}`